### PR TITLE
Clean up "c_string_type" and "c_string_encoding" for Py3

### PR DESCRIPTION
**Pull request by scoder**
Make "c_string_type=str" a synonym of "c_string_type=unicode" and "c_string_encoding=default" a synonym of "c_string_encoding=utf8".

Closes https://github.com/cython/cython/issues/6280

### CODE REVIEW

File: docs/src/tutorial/strings.rst [hunk 1]
```diff
@@ -381,40 +381,30 @@ return empty strings for out-of-bounds indices).
 Auto encoding and decoding
 --------------------------
 
-Cython 0.19 comes with two new directives: ``c_string_type`` and
+Automatic conversions is controlled by the directives ``c_string_type`` and
```
Comment (da-woods) [inline docs/src/tutorial/strings.rst:384]:
```suggestion
Automatic conversions are controlled by the directives ``c_string_type`` and
```

**Comment by scoder** [comment]
>Automatic conversions are controlled by the directives

I actually meant singular (conversion). Which do you think is better?

**Comment by da-woods** [comment]
> Automatic conversions are controlled by the directives
> I actually meant singular (conversion). Which do you think is better?

Probably "conversions" but not a hugely strong preference

**Comment by scoder** [comment]
The crashes in Py3.13 are due to the release of Py3.13b3 and also appear in the master branch.

**Comment by da-woods** [comment]
> The crashes in Py3.13 are due to the release of Py3.13b3 and also appear in the master branch.

I can't reproduce them locally unfortunately. (I do agree with your assessment though - they appeared exactly when b3 appeated)

**Pull request merged**